### ISSUE TITLE
Test Email Fails, Campain Works with same settings

### ISSUE THREAD

**Issue by skymoore** [knadh/listmonk#1450]
**Version:**
 - listmonk: [eg: v2.5.0]
 - OS: Official Docker Image

**Description of the bug and steps to reproduce:**
Protonmail SMTP authentication fails when trying to send a test email with the correct credentials, and authentication mechanism, [correct settings for protonmail smtp found here](https://proton.me/support/smtp-submission).  When sending a campaign it works fine.

Not sure why the server restarted as shown in logs, but still can't send a test email with credentials that work for a campaign.

**Logs:**
```
2023/08/13 01:01:05 settings.go:288: error sending SMTP test (Test connection): 535 5.7.8 Error: authentication failed: authentication failed
2023/08/13 01:01:54 settings.go:288: error sending SMTP test (Test connection): 535 5.7.8 Error: authentication failed: authentication failed
2023/08/13 01:01:56 init.go:822: reloading on signal ...
2023/08/13 01:01:56 init.go:791: HTTP server shut down
2023/08/13 01:01:56 main.go:102: v2.5.0 (ea2f93e 2023-08-10T18:36:19Z, linux/amd64)
2023/08/13 01:01:56 init.go:145: reading config: config.toml
2023/08/13 01:01:56 init.go:273: connecting to db: pgsql-postgresql-ha-pgpool.postgresql-ha.svc.cluster.local:5432/listmonk
2023/08/13 01:01:57 init.go:593: media upload provider: filesystem
2023/08/13 01:01:57 init.go:517: loaded email (SMTP) messenger: noreply@redacted@smtp.protonmail.ch
⇨ http server started on [::]:9000
2023/08/13 01:15:52 manager.go:507: start processing campaign (test)
2023/08/13 01:15:52 manager.go:686: campaign (test) finished
```

**Comment by knadh** [comment]
I think this may be the issue - https://github.com/knadh/listmonk/pull/1453

Are you sending a test mail from the campaign view or the Settings->SMTP test. If it's the latter, can you manually clear the password in the SMTP config, re-enter the password in the password field and try the test e-mail?

**Comment by skymoore** [comment]
I am sending from Settings->SMTP test.  I will spin it back up and try again.  I was experiencing some other issues with 2.5.0 as well, like when I would add a list I would get an error that says list does not exist, but then when I would refresh the page the list would be there, same for campaigns.  Pretty basic setup all with environment variables.  Only thing I'm unsure about is compatibility with postgres-15.  You know anything about that?  Also is there any way to increase verbosity of logging?  Seeing errors in the web-ui, and nothing in the console at all is really frustrating.

**Comment by knadh** [comment]
It's aforementioned bug in the UI warning then.

> Seeing errors in the web-ui, and nothing in the console at all is really frustrating.

All errors that come from the backend that are shown as toasts are logged on stdout and also in Settings -> Logs

**Comment by StipoMargic** [comment]
Have problem with smtp after updating to 2.5.0

Is there anu wau to downgrade?

**Comment by MaximilianKohler** [comment]
> Have problem with smtp after updating to 2.5.0
> 
> Is there anu wau to downgrade?

What problem?

**Comment by StipoMargic** [comment]
Testing SMTP in Settings fails with message that I am unauth retyping password dose not help even with other accounts.

**Comment by MaximilianKohler** [comment]
>retyping password dose not help 

This is strange, but the other bug was fixed in v2.5.1. Try that and see.

**Comment by StipoMargic** [comment]
Seems it works in 2.5.1. ty
